### PR TITLE
fix: update templates with new version

### PR DESCRIPTION
**Pull request by julianiff**

### CODE REVIEW

**Review by ragi96 — COMMENTED** [review]
Misses `changeset`

**Review by marguslant0 — APPROVED** [review]

File: .changeset/strange-rice-juggle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
+---
+
+fix: update template dependencies
```
Comment (ragi96) [inline .changeset/strange-rice-juggle.md:5]:
```suggestion
chore: bump dependency `@frontify/frontify-cli` to `^5.6.19` for the templates
```

Wdyt?

Comment (julianiff) [reply .changeset/strange-rice-juggle.md:5]: sure thing, i still have some thing to update, ill add them then 👍

Comment (julianiff) [reply .changeset/strange-rice-juggle.md]: 🏁

File: packages/cli/templates/content-block-tailwind/README.md [hunk 2]
```diff
@@ -1 +1,43 @@
-# Frontify Block Boilerplate
+# Hello World
+This is a simple example of a Hello World app using React and Tailwind.
```
Comment (ragi96) [inline packages/cli/templates/content-block-tailwind/README.md:2]: I think this one is in the wrong folder, as its descibes a folder but this is a block

File: .changeset/strange-rice-juggle.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
+---
+
+chore: bump dependency `@frontify/frontify-cli` to `^5.6.19` for the templates and `@frontify/app-bridge` to `4.0.0-alpha.29`.
```
Comment (ragi96) [inline .changeset/strange-rice-juggle.md:5]: misses fondue

File: packages/cli/templates/content-block-tailwind/README.md (hunk 2 above)
Comment (julianiff) [reply packages/cli/templates/content-block-tailwind/README.md]: haha indeed, sorry

Comment (ragi96) [reply packages/cli/templates/content-block-tailwind/README.md]: np all good now :)

**Review by ragi96 — APPROVED** [review]

**Pull request merged**